src/components/WalletDropdown/LanguageMenu.tsx

```diff
@@ -29,6 +29,8 @@ const InternalLinkMenuItem = styled(InternalMenuItem)<{ isActive: boolean }>`
   color: ${({ theme }) => theme.textPrimary};
   :hover {
     cursor: pointer;
+    background-color: ${({ isActive, theme }) => !isActive && theme.backgroundModule};
```

Comment:
I disabled the hover state on the active language.  I thought it looked really weird imo and technically since it's selected it is not actionable

Reply:
is there an `:active` element you could use instead, or is that unrelated?

Reply:
this is unrelated styling.  By active, I mean the users selected language, not when they are clicking on it